### DISCUSSION TITLE
Full size emoji in a message without text

### DISCUSSION THREAD

**Discussion by jeremybigot** [flyerhq/flutter_chat_ui#121] [Ideas & Requests]
It would be great if we could have a full size emoji (without the background) if the message contains only an emoji and not a text like Telegram or Whats'App

<img width="277" alt="Capture d’écran 2021-09-27 à 17 21 17" src="https://user-images.githubusercontent.com/5739662/134937715-7ea47ebc-5dae-412d-b433-d64002f89c50.png">

**Comment by demchenkoalex** [comment]
Hi Jérémy, this is something we would love to have too, but ATM we are all busy

**Reply by jeremybigot** [reply]
Hi, No problem, I'll wait patiently then!

**Reply by lucasjinreal** [reply]
@demchenkoalex Hi, this indeed a must-have feature for a Chat app, since facebook messager and iMessager all support this. 

Ideally, seems not hard to implement

**Reply by demchenkoalex** [reply]
It actually was implemented and is available

**Comment by demchenkoalex** [comment]
Available since v1.5.0

**Reply by lucasjinreal** [reply]
I will have a try
